### PR TITLE
[01]funds-page

### PR DESCRIPTION
**Pull request by ChabuChabu**
Funds form page

top
Top
Funds bottom
funds-bottom

### CODE REVIEW

File: assets/js/business-usage-form.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+}
+function fundForm (e){
+    required(firstUse, e);
+    required (firstUseSummary, e);
+    required (secondUse, e);
+    required (secondUseSummary, e);
+    required (thirdUse, e);
+    required (thirdUseSummary, e);
+    required (ForthUse ,e);
```
Comment (tundeojediran) [inline assets/js/business-usage-form.js]: camel casing forthUse instead of ForthUse

File: forms/business-usage-form.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    <!--Custom Styles-->
+    <link rel="stylesheet" type="text/css" href="../assets/styles/businessFunds.css">
+    <!--Font owesomes-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Business Usage</title>
+</head>
+<body>
+  
+   <!--page wrapper-->
+   <div class="paga-wrapper">
```
Comment (tundeojediran) [inline forms/business-usage-form.html]: do you mean 'page-wrapper'

File: assets/js/business-usage-form.js (hunk 1 above)
Comment (ChabuChabu) [reply assets/js/business-usage-form.js]: Resolved

File: forms/business-usage-form.html (hunk 2 above)
Comment (ChabuChabu) [reply forms/business-usage-form.html]: Yes updated

**Review by faithakinbobola — APPROVED** [review]

**Pull request merged**